website/blog/2020-08-04-2.1.0.md

```diff
@@ -0,0 +1,2241 @@
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+class a extends a // comment
+{
+  constructor() {}
+}
+
+// Prettier stable
+class a extends a // comment {
+  constructor() {}
+}
+
+// Prettier master
+class a extends a { // comment
+  constructor() {}
+}
+```
+
+#### Don't wrap arrays in Jest `test.each` template strings ([#8354](https://github.com/prettier/prettier/pull/8354) by [@yogmel](https://github.com/yogmel))
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+test.each`
+a | b | c
+${1} | ${[{ start: 1, end: 3 },{ start: 15, end: 20 },]} | ${[]}
+`("example test", ({a, b, c}) => {})
+
+// Prettier stable
+test.each`
+  a | b | c
+  ${1} | ${[
+  { start: 1, end: 3 },
+  { start: 15, end: 20 }
+]} | ${[]}
+`("example test", ({ a, b, c }) => {});
+
+// Prettier master
+test.each`
+  a    | b                                                 | c
+  ${1} | ${[{ start: 1, end: 3 }, { start: 15, end: 20 }]} | ${[]}
+`("example test", ({ a, b, c }) => {});
+```
+
+#### Support `insertPragma` and `requirePragma` in files has shebang ([#8376](https://github.com/prettier/prettier/pull/8376) by [@fisker](https://github.com/fisker))
+
+<!-- prettier-ignore -->
+```js
+// `--insert-pragma`
+// Input
+#!/usr/bin/env node
+hello
+  .world();
+
+// Prettier stable
+SyntaxError: Unexpected token (3:1)
+  1 | /** @format */
+  2 |
+> 3 | #!/usr/bin/env node
+    | ^
+  4 | hello
+  5 |   .world();
+
+// Prettier master
+#!/usr/bin/env node
+/** @format */
+
+hello.world();
+
+// `--require-pragma`
+// Input
+#!/usr/bin/env node
+/**
+ * @format
+ */
+hello
+  .world();
+
+// Prettier stable
+#!/usr/bin/env node
+/**
+ * @format
+ */
+hello
+  .world();
+
+// Prettier master
+#!/usr/bin/env node
+/**
+ * @format
+ */
+hello.world();
+```
+
+#### Fix range formatting indentation ([#8410](https://github.com/prettier/prettier/pull/8410) by [@thorn0](https://github.com/thorn0))
+
+```console
+> echo -e "export default class Foo{\n/**/\n}" | prettier --range-start 16 --range-end 31 --parser babel
+```
+
+<!--prettier-ignore-->
+```js
+// Prettier stable
+export default class Foo {
+                 /**/
+               }
+
+// Prettier master
+export default class Foo {
+  /**/
+}
+```
+
+#### Improve detection of source elements for range formatting ([#8419](https://github.com/prettier/prettier/pull/8419) by [@thorn0](https://github.com/thorn0))
+
+Not all statement types were detected (read [here](https://prettier.io/docs/en/options.html#range) how range formatting works in Prettier).
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+for   (const  element of list) { /* ... */ }
+//     ^^^^^^^^^^^^^^^^^^^^^^ ← range
+
+// Prettier stable
+for   (const  element of list) { /* ... */ }
+
+// Prettier master
+for (const element of list) {
+  /* ... */
+}
+```
+
+#### Support Private Fields in `in` ([#8431](https://github.com/prettier/prettier/pull/8431) by [@sosukesuzuki](https://github.com/sosukesuzuki))
```

Comment:
only on Babel parsers, right?

(I did just raise PR #8942 to run the test on all Babel parsers.)

Reply:
Yes, works on only babel